samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -384,4 +395,15 @@ private TaskFactory createTaskFactory() {
           new EndOfStreamMessage(null)));
       });
   }
+
+  /**
+   * Only adds a config from {@code config} to samza job {@code configs} if they dont exist in it.
+   * @param config for the application
+   * @return this {@link TestRunner}
+   */
+  private TestRunner putIfAbsentConfig(Map<String, String> config) {
```

Comment:
Don't need a helper method for a one line method, can inline this.

Reply:
@Sanil15 this is related to the type-erasure of generic types. If you declare descriptor as InMemoryInputDescriptor&lt;?&gt;, the one liner would work. Otherwise, since the descriptor is of an erased type InMemoryInputDescriptor, Java will complain about type conversion failure in compile time.